diff --git a/node_modules/swagger-express-ts/dist/api-model-property.decorator.js b/node_modules/swagger-express-ts/dist/api-model-property.decorator.js
index 1955de7..aa753af 100644
--- a/node_modules/swagger-express-ts/dist/api-model-property.decorator.js
+++ b/node_modules/swagger-express-ts/dist/api-model-property.decorator.js
@@ -4,7 +4,7 @@ var swagger_service_1 = require("./swagger.service");
 function ApiModelProperty(args) {
     return function (target, propertyKey) {
         var propertyType = Reflect.getMetadata("design:type", target, propertyKey)
-            .name;
+            ?.name;
         swagger_service_1.SwaggerService.getInstance().addApiModelProperty(args, target, propertyKey, propertyType);
     };
 }
